modules/@apostrophecms/schema/ui/apos/mixins/AposInputMixin.js

```diff
@@ -19,7 +19,8 @@ export default {
         return [];
       },
       type: Array
-    }
+    },
+    triggerValidation: Boolean
```

Comment:
This should be formatted like the others with `default: false`.

Reply:
Done